### PR TITLE
Fix detection of dashboard build files in installed packages.

### LINKED ISSUE #1083: cli fails on google colab
Example of failure: https://colab.research.google.com/drive/1WqSqLg-o55S1Fj3Og8_7I0B4M2G7NchK?usp=sharing

```
Traceback (most recent call last):
  File "/usr/local/bin/orion", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/orion/core/cli/__init__.py", line 34, in main
    load_modules_parser(orion_parser)
  File "/usr/local/lib/python3.8/dist-packages/orion/core/cli/__init__.py", line 19, in load_modules_parser
    modules = module_import.load_modules_in_path(
  File "/usr/local/lib/python3.8/dist-packages/orion/core/utils/module_import.py", line 33, in load_modules_in_path
    return list(modules)
  File "/usr/local/lib/python3.8/dist-packages/orion/core/utils/module_import.py", line 28, in <lambda>
    modules = map(lambda f: __import__(path + "." + f, fromlist=[""]), files)
  File "/usr/local/lib/python3.8/dist-packages/orion/core/cli/frontend.py", line 58, in <module>
    class StaticResource:
  File "/usr/local/lib/python3.8/dist-packages/orion/core/cli/frontend.py", line 61, in StaticResource
    STATIC_DIR = get_dashboard_build_path()
  File "/usr/local/lib/python3.8/dist-packages/orion/core/cli/frontend.py", line 51, in get_dashboard_build_path
    raise RuntimeError(
RuntimeError: Cannot find dashboard static files to run frontend. Expected to be located at: /usr/orion-dashboard/build
```

It looks like the `build` of the dashboard is not included in the files when installing on the jupiter notebook.

### PR DESCRIPTION
**Pull request by notoraptor**
# Description

Hi @bouthilx ! This PR tries to fix #1083 . However, I don't know how to test it in CI. CI currently tests normal installation, but this issue seems related to a corner case, and I don't know how to reproduce it in CI.

# Changes

t seems that, depending on installation, additional files may be installed in `<sys.prefix>/local` instead of just `<sys.prefix>`. More info: https://stackoverflow.com/questions/14211575/any-python-function-to-get-data-files-root-directory#comment99087548_14211600

So, this PR just checks both `<sys.prefix>` then `<sys.prefix>/local` to find dashboard build parent folder (named `orion-dashboard`).

On regular installations (e.g. in CI tests), checking `<sys.prefix>` is enough. But it seems data are installed in `<sys.prefix>/local` on Jupyter installations. I don't know why there's this difference.

# Checklist

## Tests
- [ ] I added corresponding tests for bug fixes and new features. If possible, the tests fail without the changes
- [ ] All new and existing tests are passing (`$ tox -e py38`; replace `38` by your Python version if necessary)

## Documentation
- [ ] I have updated the relevant documentation related to my changes

## Quality
- [ ] I have read the [CONTRIBUTING](https://github.com/Epistimio/orion/blob/develop/CONTRIBUTING.md) doc
- [ ] My commits messages follow [this format](https://chris.beams.io/posts/git-commit/)
- [ ] My code follows the style guidelines (`$ tox -e lint`)

### CODE REVIEW

**Comment by notoraptor** [comment]
@bouthilx it seems to have worked, including `conda-build` !

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -287,6 +287,18 @@ jobs:
           if ( ls ${ORION_PREFIX}/orion-dashboard/build/static/js/main.*.js ); then true; else false; fi
           echo Check if frontend script can find dashboard build
           python -c "from orion.core.cli.frontend import get_dashboard_build_path; get_dashboard_build_path();"
+
+          echo Check if frontend script can find dashboard build on sys.prefix/local
```
Comment (bouthilx) [inline .github/workflows/build.yml:291]: Did you verify that this fails as expected without the fix?

Comment (notoraptor) [reply .github/workflows/build.yml:291]: Hello @bouthilx ! Yes, I can confirm it fails as expected when using current PYPI package (`pip install orion`). But I don't know how to put it in CI tests.

Comment (bouthilx) [reply .github/workflows/build.yml:291]: It's good enough to have it in the build here.

**Review by bouthilx — APPROVED** [review]
